### PR TITLE
add apps.py files to account, app and util and specify default_app_co…

### PR DESCRIPTION
**Pull request by jake-kent**
…nfig in each app's __init__.py

### CODE REVIEW

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/app/apps.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+class MainAppConfig(AppConfig):
```
Comment (jschneier) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/app/apps.py]: I want to de-emphasize (and probably eventually remove) ``app``. You can call this `AppConfig` so long as your alias the import:
```python
from django.apps import AppConfig as DjangoAppConfig

class AppConfig(DjangoAppConfig):
  ...
```

I actually think we should rename this to `home` pursuant to @gone suggestion.

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/__init__.py [hunk 2]
```diff
@@ -1 +1 @@
-
+default_app_config = 'app.apps.MainAppConfig'
```
Comment (jschneier) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/__init__.py]: I think this needs to be `{{ cookiecutter.repo_name }}.app.apps.MainAppConfig` but confirmation would be nice.

Comment (jake-kent) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/__init__.py]: Already within the first {{ cookiecutter.repo_name }} folder, so not necessary.

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/home/apps.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig as DjangoAppConfig
+
+class HomeConfig(DjangoAppConfig):
+	name = "main_app"
```
Comment (gone) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/home/apps.py]: Should be home

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/home/management/commands/loadcsv.py [hunk 4]
```diff
@@ -1,6 +1,6 @@
 from django.core.management.base import BaseCommand, CommandError
 
-#from app.models import SiteExperience
+#from home.models import SiteExperience
```
Comment (gone) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/home/management/commands/loadcsv.py]: pull this line out

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/__init__.py [hunk 5]
```diff
@@ -1 +1 @@
-
+default_app_config = 'home.apps.MainAppConfig'
```
Comment (gone) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/__init__.py]: I think we don't want to use this - 
https://docs.djangoproject.com/en/1.9/ref/applications/#configuring-applications

New applications should avoid default_app_config. Instead they should require the dotted path to the appropriate AppConfig subclass to be configured explicitly in INSTALLED_APPS.

https://docs.djangoproject.com/en/1.9/ref/applications/#configuring-applications

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/home/urls.py [hunk 6]
```diff
@@ -7,5 +7,3 @@
     url(r'^error/', error, name='error'),
     url(r'^$', TemplateView.as_view(template_name="index.html"), name="home"),
 ]
-
-from .signals import * #ensure that the signals are attatched via import
```
Comment (gone) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/home/urls.py]: I think this should be imported in the app config

https://docs.djangoproject.com/en/1.9/ref/applications/#django.apps.AppConfig.ready

**Pull request merged**